### PR TITLE
Add Viber messaging channel

### PR DESCRIPTION
**Pull request by hebime**
This adds a Viber section to the docs.
Right now there's no process to get a public account token since it's done by contacting Viber directly.

@dannytranlx @alavers @mspensieri @wmora @jpjoyal

### CODE REVIEW

**Comment by hebime** [comment]
[[Viber, docs] Doc update](https://trello.com/c/ebMa6f7b/4676-viber-docs-doc-update)

**Comment by hebime** [comment]
😱  lol

**Comment by hebime** [comment]
[[Viber, docs] Doc update](https://trello.com/c/ebMa6f7b/4676-viber-docs-doc-update)

File: source/includes/messaging_channels/_viber.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Viber
+
+Our Viber integration allows users to send you messages through a Viber Public Account. You will receive these messages in one of our other channels like Slack, HipChat or Help Scout.
+Your replies will be sent back to user through the Viber Public Account.
```
Comment (jpjoyal) [inline source/includes/messaging_channels/_viber.md]: "back to user" -> "back to users" no ?

File: source/includes/messaging_channels/_viber.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+### Configuring Viber
+
+Once you have your Public Account token, copy and paste it into the Public Account token box on the Viber configuration page.
```
Comment (jpjoyal) [inline source/includes/messaging_channels/_viber.md]: Should add a sentence about finding viber in the integration directory

Comment (alavers) [inline source/includes/messaging_channels/_viber.md]: or better yet a link directly to it

Comment (alavers) [inline source/includes/messaging_channels/_viber.md]: A link to Viber public account docs either here, or directly in the integration UI would be handy.

Comment (hebime) [inline source/includes/messaging_channels/_viber.md]: I believe those docs are still a work a in progress on Viber's end. Once they finish them, we'll add them here.

File: source/includes/messaging_channels/_viber.md (hunk 1 above)
Comment (hebime) [inline source/includes/messaging_channels/_viber.md]: yes!

File: source/includes/messaging_channels/_viber.md (hunk 2 above)
Comment (hebime) [inline source/includes/messaging_channels/_viber.md]: Added a link to the integration page.

File: source/includes/messaging_channels/_viber.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Viber
+
+Our [Viber integration](https://app.smooch.io/integrations/viber) allows users to send you messages through a Viber Public Account. You will receive these messages in one of our other channels like Slack, HipChat or Help Scout.
+Your replies will be sent back to users through the Viber Public Account.
+
+### Configuring Viber
+
+Once you have your Public Account token, copy and paste it into the Public Account token box on the Viber configuration page.
+Once you paste your token into the box, click on "Connect your Viber Public Account".
```
Comment (jpjoyal) [inline source/includes/messaging_channels/_viber.md]: nit: remove the double "once" by doing this change  [...] token box on the Viber configuration page then click on "Connect your Viber Public Account".

**Comment by jpjoyal** [comment]
rest looks good

File: source/includes/messaging_channels/_viber.md (hunk 3 above)
Comment (hebime) [inline source/includes/messaging_channels/_viber.md]: 👍

**Pull request merged**